pom.xml

```diff
@@ -374,6 +374,11 @@ under the License.
           <artifactId>maven-shade-plugin</artifactId>
           <version>3.2.4</version>
         </plugin>
+        <plugin><!-- TODO remove when upgrading parnet POM to 35 -->
```

Comment:
Could _parnet_ here and above be fixed, please?

Reply:
fixed, tx